app/controllers/api/v1/packages_controller.rb

```diff
@@ -251,6 +260,10 @@ def stock_serializer
         Api::V1::StockitItemSerializer
       end
 
+      def not_splittable?(qty_to_split)
```

Comment:
I like the idea of having this method on the model so that a package can tell us if it is splittable or not. Also useful method to have outside of the controller. Maybe move to `@package.splitable?`

Reply:
I agree with it 👍